webapp/actions/global_actions.jsx

```diff
@@ -593,3 +594,15 @@ export function emitPopoverMentionKeyClick(isRHS, mentionKey) {
         mentionKey
     });
 }
+
+let serverVersion = '';
```

Comment:
Should this be initialized to `Client4.getServerVersion()` to avoid a refresh the first time `reloadIfServerVersionChanged()` is called? Seems to me that because `serverVersion` is initialized to an empty string, the if statement on line 602 will always be true the first time that function is called, and cause an unnecessary refresh.